core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -184,6 +189,12 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
           error("whitelist must be specified when using new consumer in mirror maker.")
           System.exit(1)
         }
+
+        println("WARNING: The default partition assignment strategy of the new-consumer-based mirror maker will " +
+          "change from 'range' to 'roundrobin' in the future (so that a better load balancing can be achieved). " +
+          "One easy way to keep using your current default strategy ('range') is to add the following to the new " +
```

Comment:
I feel it would probably be more helpful here to describe how the user can change to 'round-robin' in advance of the upcoming change (there really shouldn't be any need for users to stick with 'range'). Also, maybe we should only print this message if the user has not overridden the assignment strategy?